converter.py

```diff
@@ -0,0 +1,29 @@
+import os
+import hashlib
+from os import walk
+import mimetypes
+import json
+import shutil
+
+
+class Converter:
+    def __init__(self):
+        self.meta = {}
+
+    def convert(self, path):
+        f = []
```

Comment:
Can we give it a better name, maybe `files` ?

Reply:
Sure :)